x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -122,19 +139,25 @@ export class LoginPage extends Component<Props, State> {
       ['loginWelcome__contentDisabledForm']: !loginIsSupported,
     });
 
+    const customLogo = this.state.customBranding?.logo;
+    const logo = customLogo ? (
+      <img src={customLogo} width="40" height="40" alt="logo" />
+    ) : (
+      <EuiIcon type="logoElastic" size="xxl" />
+    );
+    const pageTitle = this.state.customBranding?.pageTitle || 'Elastic';
```

Comment:
This would literally be the page title in the home tab:
<img width="174" alt="Screenshot 2023-01-23 at 14 01 20" src="https://user-images.githubusercontent.com/1937956/214046229-d4c1382a-321c-40c3-8c8e-e84ca270a5bb.png">

Fair point that it's more like company name. I can change it for the `login_page`, but need to think about if it's worth changing in the custom branding.

Reply:
> I can change it for the login_page, but need to think about if it's worth changing in the custom branding.

Sounds good to me, thanks.

Reply:
Looks like we don't need this at all on the home page, as product is ok with keeping just `Elastic` in the messages.